do_snowDA.sh

```diff
@@ -78,17 +80,17 @@ module list
 INCDATE=${SCRIPTDIR}/incdate.sh
 
 # substringing to get yr, mon, day, hr info
-YYYY=`echo $THISDATE | cut -c1-4`
-MM=`echo $THISDATE | cut -c5-6`
-DD=`echo $THISDATE | cut -c7-8`
-HH=`echo $THISDATE | cut -c9-10`
+export YYYY=`echo $THISDATE | cut -c1-4`
+export MM=`echo $THISDATE | cut -c5-6`
+export DD=`echo $THISDATE | cut -c7-8`
+export HH=`echo $THISDATE | cut -c9-10`
 
 PREVDATE=`${INCDATE} $THISDATE -6`
 
-YYYP=`echo $PREVDATE | cut -c1-4`
-MP=`echo $PREVDATE | cut -c5-6`
-DP=`echo $PREVDATE | cut -c7-8`
-HP=`echo $PREVDATE | cut -c9-10`
+export YYYP=`echo $PREVDATE | cut -c1-4`
+export MP=`echo $PREVDATE | cut -c5-6`
+export DP=`echo $PREVDATE | cut -c7-8`
+export HP=`echo $PREVDATE | cut -c9-10`
```

Comment:
Actually this might become a problem, if the DA window length differs from the forecast length (which it might need to for JEDI reasons). But, if that happens the better solution would be to change the variable name here. I might change this later.

Reply:
Agree with you. I did have this problem when I conducted the GTS snow DA by setting the DA window length as 8 hours in order to use all observations in that cycle.